### ISSUE TITLE
Motion principles - accessibility considerations

### ISSUE THREAD

**Issue by kittybourne** [telus/tds-community#249]
## Problem statement

- As a designer or developer, I would like to create animations that are accessible for all users, so that I can be consistent with TELUS design system.

## Recommendation

- This is not a TDS component. Just something I want to run by DPA and get alignment before we update standards
- Motion principles are now published in BRC however as identified by the accessibility team it is missing some important accessibility considerations and decisions the team can make about fallbacks and take over experiences such as parallax and scroll jacking.

## Design intent

No designs needed, just update to standards copy.

## Acceptance criteria

Draft copy to start with:

*Accessibility considerations*

Be conservative about the use of animations especially if it affects a large part of the screen, it can visually disorientating and can cause people to feel sick.

User initiated animation is much better received for system status and visual feedback and learning spatial relationships on a site. Avoid scroll jacking, parallax and autoplay videos. We should allow the user to control the experience on screen. If there are automatic animations you must provide the user with the option to turn to it off.

There are built in controls in the operating system preferences to reduce motion on devices and on desktop. If the user has this enabled, the fall back is to make it static. To do so you can use a media query. Here is an example:

@media (prefers-reduced-motion: reduce) {

const backgroundAnimation = keyframes`
 from { opacity: 0; }
 to { opacity: 1; }
`;

const ScrollBackgroundAfter = styled.div`
 opacity: 0;
 animation: ${backgroundAnimation} 0.2s;

 @media (prefers-reduced-motion: reduce) {
   opacity: 1;
 }
`;
}

Animations should use 30 frames per sec to make it as seamless as possible. The human eye sees at a rate of about 200 frames per second.

**Comment by varunj90** [comment]
Kitty to move forward with updating Motion principles and get back to DPA before going live on BRC

**Comment by varunj90** [comment]
Update: Chantel Chyb is working on the content with Beth, along with the translations

Kitty will review with DPA once ready!

**Comment by kittybourne** [comment]
Copy here: https://docs.google.com/document/d/1svRB7kUVViefVbPn47627mqKqW5JQS4qZsSbzMkpLvw/edit
